cfgov/cfgov/settings/base.py

```diff
@@ -650,4 +650,4 @@
 
 # If server's clock drifts from NTP by more than specified offset
 # (in seconds), check_clock_drift will fail
-MAX_ALLOWED_TIME_OFFSET = 5
+MAX_ALLOWED_TIME_OFFSET = 5
```

Comment:
No idea why this is showing up in the diff.

Reply:
Could your editor be set to remove trailing new lines automatically?